node-init-image-cache/cloudbuild-install-cronjob.yaml

```diff
@@ -75,7 +75,7 @@ steps:
           sed -e 's/$${PROJECT_ID}/${PROJECT_ID}/g' \
               -e 's/$${_PROVISION_MACHINE_TYPE}/${_PROVISION_MACHINE_TYPE}/g' \
               -e 's/$${_IMAGE_BUILD_REGION}/${_IMAGE_BUILD_REGION}/g' \
-              -e 's/$${_IMAGE_BUILD_ZONE}/$${BUILD_ZONE}/g' \
```

Comment:
I think it’s just the single quotes blocking bash substitution and you could fix it with just this:

-e 's/$${_IMAGE_BUILD_ZONE}/'$${BUILD_ZONE}'/g' \

Reply:
@danisla Thanks for the recommendation. I just updated the PR with the suggested change.